optimade/models/structures.py

```diff
@@ -46,43 +46,45 @@ class StructureFeatures(Enum):
 
 class Species(BaseModel):
     """A list describing the species of the sites of this structure.
-    Species can be pure chemical elements, or virtual-crystal atoms representing a statistical occupation of a given site by multiple chemical elements.
+Species can represent pure chemical elements, virtual-crystal atoms representing a
```

Comment:
Shouldn't this be indented, or is there an issue with rendering the documentation here if so?

Reply:
This is where the issue comes in, yeah. If there's a list in the docstring, it doesn't like it being indented, so I've just dedented all those cases.